src/components/BorrowerProfile/SummaryCard.vue

```diff
@@ -42,9 +42,31 @@
 					</div>
 				</template>
 				<template v-else>
+					<a
+						href="#bp-comments-jump-link"
+						class="tw-text-black hover:tw-text-white kvpill-wrapper"
+						v-kv-track-event="[
+							'borrower-profile',
+							'click',
+							'jump-link',
+							'comments-pill'
+						]"
+					>
+						<kv-pill
```

Comment:
This is similar to the KvCallout... not sure if you can override the bg color on that though.
https://main--608b4cf87f686c00213841b1.chromatic.com/?path=/story/kvloancallouts--default

So it "might" be usable, though it would be good to have something more reusable for this pill style.

Where did this BG color come from? Why?

Reply:
Yeah its kind of like KvCallout, I think it's more like the other component on the BP, the SummaryTag component. I will combine KvPill and Summary tag into 1, since the only change would be the background color.

Reply:
These blues are one-off colors 😖

Reply:
colors lolz ;) If you see opportunity to an appropriate kv-component please create a ticket and go for it, but it doesn't need to block your work here, could just be something to build and swap in once ready.